<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Desk215" failures="1" tests="4" name="StepDefinition.CucumberTestRunner" time="54.230" errors="0" timestamp="2024-07-10T17:20:19 IST" skipped="0">
  <testcase classname="StepDefinition.CucumberTestRunner" name="runScenario" time="12.560"/>
  <testcase classname="StepDefinition.CucumberTestRunner" name="runScenario" time="17.256">
    <failure message="user is navigated to  expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: user is navigated to  expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at StepDefinition.LoginSteps.user_is_navigated_to_the_home_page(LoginSteps.java:54)
at âœ½.user is navigated to the home page(file:///C:/Users/mohamedikfan.m/Desktop/SELENIUM/CucumberProject/src/test/resources/Features/login.feature:11)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="StepDefinition.CucumberTestRunner" name="runScenario" time="9.868"/>
  <testcase classname="StepDefinition.CucumberTestRunner" name="runScenario" time="14.546"/>
</testsuite> <!-- StepDefinition.CucumberTestRunner -->
